inst/BS5/assets/pkgdown.scss

```diff
@@ -121,12 +136,6 @@ aside {
 }
 
 /* table of contents -------------------------------------------------------- */
-$pkgdown-toc-border-width: 0 !default;
```

Comment:
No evidence that any one has used these: https://github.com/search?q=pkgdown-toc+&type=code